include/seastar/rpc/rpc.hh

```diff
@@ -203,7 +203,11 @@ class logger {
 
     // _seastar_logger will always be used first if it's available
     template <typename... Args>
+#if FMT_VERSION >= 80000 && defined(SEASTAR_DEBUG)
```

Comment:
Why SEASTAR_DEBUG? Since it's compile time it should work in any mode, no?

Reply:
i dropped the `SEASTAR_DEBUG` check in the latest revision.